variables.tf

```diff
@@ -0,0 +1,110 @@
+variable "cluster_arn" {
+  type        = string
+  description = "The ARN of the ECS cluster"
+  default     = null
+}
+
+variable "config" {
+  description = "A map containing all labeler configuration options"
+  type = object({
```

Comment:
Are these all optional?

Reply:
I could not find it in the cli what is and what isn't